src/const/urls/index.ts

```diff
@@ -0,0 +1,3 @@
+export const urls = {
+  DEFAULT_PROFILE: 'https://image.bcsdlab.com/default-profile.png',
+};
```

Comment:
위와 겹치는 내용이지만, 저는 이게 더 깔끔한 것 같은데, 객체로 사용하신 이유가 있나요?
```suggestion
export const DEFAULT_PROFILE = 'https://image.bcsdlab.com/default-profile.png';
```

Reply:
확장성을 고려하여 객체로 사용했습니다.
추후 상수처럼 사용될 urls 가 늘어나면 `urls.사용할 url` 처럼 사용하고자 했습니다.

Reply:
마이너하지만, 참고하면 괜찮을 내용을 가져와봤습니다.
저희가 따로 코드 컨벤션을 정하진 않았지만, 개인적으로는 Airbnb에서 사용하는 [코드 컨벤션](https://github.com/airbnb/javascript?tab=readme-ov-file#naming--uppercase)을 자주 참고합니다.
상수의 경우 이렇게 사용하는 것도 괜찮을 거 같아요 (e.g. `EXPORTED_OBJECT.key`)
```suggestion
export const URLS = {
  defaultProfile: 'https://image.bcsdlab.com/default-profile.png',
};
```

Reply:
예전에 우테코 할 때 기억으로 했었는데 반대로 기억하고있었네요 ㅋㅋㅋㅋㅋㅋ
Airbnb 컨벤션에 맞춰 상수를 정의하도록 수정했습니다. e78a71489e574ad4852d324cbbcfb5ff8f5db4b3